server/base/src/main/java/org/apache/accumulo/server/ServerContext.java

```diff
@@ -132,6 +134,7 @@ private ServerContext(ServerInfo info) {
     securityOperation =
         memoize(() -> new AuditedSecurityOperation(this, SecurityOperation.getAuthorizor(this),
             SecurityOperation.getAuthenticator(this), SecurityOperation.getPermHandler(this)));
+    jvmGcLogger = memoize(() -> new GarbageCollectionLogger(this));
```

Comment:
The local name `jvmGcLogger` is better but I think the name of this class is confusing. Some reference to Java or the JVM in the name would greatly help to differentiate it from the Accumulo GC classes. Something like `JVMGcLogger` or `JavaMemoryLogger` or `JavaGcLogger` would be a lot better.

Reply:
Renamed in 80f5e35834 to `JvmGcLogger`